<launch>
    <!-- Carico confini workspace -->
    <param name="/workspace/x_min" value="-1.0"/>
    <param name="/workspace/x_max" value="14.0"/>
    <param name="/workspace/y_min" value="-1.0"/>
    <param name="/workspace/y_max" value="14.0"/> 
    
    <!-- Zeno + Lidar -->
    <include file="$(find models)/launch/my_zeno.launch"/>

    <!-- Nodo di gmapping -->
    <include file="$(find models)/launch/mapping.launch"/>
    
    <!-- Nodo di navigazione -->
    <node pkg="rl_agent" type="rl_agent3actions.py" name="rl_agent" output="screen" />

    <!-- Visualizzazione workspace -->
    <node pkg="common_fun" type="workspace_pub.py" name="workspace_pub" output="screen" />

    <!-- Rviz -->
    <node pkg="rviz" type="rviz" name="rviz" output="screen" args="-d $(find models)/config/unicycle.rviz"/>
    
</launch>